src/Test/Utilities/Portable/project.json

```diff
@@ -9,16 +9,14 @@
     "System.Diagnostics.Tools": "4.0.1",
     "System.IO.FileSystem": "4.0.1",
     "System.Reflection.TypeExtensions": "4.1.0",
+    "System.Text.RegularExpressions": "4.1.0",
```

Comment:
@ericstj you might find this curious. Updating the imports section below required us to add this package, or we couldn't find the Regex type. This clearly should have been there all along, but we are unsure how it was working before.